shared/src/business/entities/WorkItem.test.js

```diff
@@ -115,4 +115,71 @@ describe('WorkItem', () => {
     workItem.assignToUser(assignment);
     expect(workItem.toRawObject()).toMatchObject(assignment);
   });
+
+  it('is set high priority if case is calendared or overridden', () => {
+    let workItem = new WorkItem(
+      {
+        assigneeId: '8b4cd447-6278-461b-b62b-d9e357eea62c',
+        assigneeName: 'bob',
+        caseStatus: CASE_STATUS_TYPES.new,
+        caseTitle: 'Johnny Joe Jacobson',
+        docketEntry: {},
+        docketNumber: '101-18',
+        docketNumberSuffix: DOCKET_NUMBER_SUFFIXES.SMALL,
+        section: DOCKET_SECTION,
+        sentBy: 'bob',
+      },
+      { applicationContext },
+    );
+    expect(workItem.highPriority).toBe(false);
+
+    workItem = new WorkItem(
```

Comment:
This expect block appears to occur twice - I think that's a mistake and we could remove one, right?

Reply:
Updated!